go.mod

```diff
@@ -38,6 +38,9 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+// TODO (@WSTARR) remove this temporary replacement of oras-go 1.2.2 with defenseunicorns version due to upgraded docker lib
+replace oras.land/oras-go v1.2.2 => github.com/defenseunicorns/oras-go v1.2.3
```

Comment:
nit, might be more readable if replace ops were at the top of the go.mod as we've done in the past.

Reply:
done